### PR TITLE
Added documentation for running test suite

### PR DESCRIPTION
**Pull request by joshuahansel**
Refs #58

### CODE REVIEW

File: doc/content/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Users will need to request access to the controlled NEAMS software from the [Nuc
 
 - [How to use the Virtual Test Bed?](resources/how_to_use_vtb.md) \\
 - [How to run a Virtual Test Bed example with the NEAMS Workbench on Sawtooth?](neams-workbench.md) \\
+- [How to run tests for the Virtual Test Bed](resources/running_vtb_tests.md) \\
```
Comment (GiudGiud) [inline doc/content/index.md:18]:
```suggestion
```
that s too technical imo for the front page

File: doc/content/resources/running_vtb_tests.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   cd ~/projects/dire_wolf
+   ```
+
+1. +Run the test script.+ The `run_tests` script must be run with the `--spec-file <vtb_dir>`
+   option, along with any other desired command-line options for `run_tests`, where
+   `<vtb_dir>` is the path to the VTB directory. For example, if the VTB is installed
+   in `~/projects/virtual_test_bed` and you want to allocate 10 processes:
+
+   ```
+   ./run_tests --spec-file ~/projects/virtual_test_bed -j 10
```
Comment (GiudGiud) [inline doc/content/resources/running_vtb_tests.md:43]: I m fine with this but this is not how i do it

i initialize and install the submodule and run the script in vtb/scripts

File: doc/content/resources/running_vtb_tests.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+you can run tests as follows:
+
+1. +Decide on the application.+ The test suite is run for a particular application.
+   If you want to test multiple applications, these steps must be followed for each.
+1. +Navigate to the application repository.+ You must change the directory
+   to the executable of interest, containing the script `run_tests`. Note that
+   it is not sufficient to call this script from another directory. For example,
+
+   ```
+   cd ~/projects/dire_wolf
```
Comment (GiudGiud) [inline doc/content/resources/running_vtb_tests.md:34]: this does not work if you dont have source access

File: doc/content/resources/running_vtb_tests.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+test can only be run by executables that meet one of the patterns `bison*`, `blue_crab*`,
+or `dire_wolf*`; if running the test suite with an incompatible executable, this
+test will be skipped.
+
+Assuming you've already followed the [directions for getting started with the VTB](resources/how_to_use_vtb.md),
+you can run tests as follows:
+
+1. +Decide on the application.+ The test suite is run for a particular application.
+   If you want to test multiple applications, these steps must be followed for each.
+1. +Navigate to the application repository.+ You must change the directory
```
Comment (GiudGiud) [inline doc/content/resources/running_vtb_tests.md:29]:
```suggestion

If you have source code access for the application:

1. +Navigate to the application repository.+ You must change the directory
```

File: doc/content/index.md (hunk 1 above)
Comment (joshuahansel) [reply doc/content/index.md]: What about maybe generalizing "Contribution guidelines to add a new model" to "How to contribute to the Virtual Test Bed" and then on that page, have my stuff as a new section?

File: doc/content/resources/running_vtb_tests.md (hunk 2 above)
Comment (joshuahansel) [reply doc/content/resources/running_vtb_tests.md:43]: I should give both options. I usually don't want to re-compile VTB's Sockeye to do my testing, so I prefer to run with my existing Sockeye.

File: doc/content/resources/running_vtb_tests.md (hunk 3 above)
Comment (joshuahansel) [reply doc/content/resources/running_vtb_tests.md:34]: True, I'll update the page accordingly

File: doc/content/index.md (hunk 1 above)
Comment (GiudGiud) [reply doc/content/index.md]: seems good to me
new page can have then:
- guidelines
- template
- testing

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**